src/NzbDrone.Core.Test/ParserTests/AbsoluteEpisodeNumberParserFixture.cs

```diff
@@ -94,6 +94,15 @@ public class AbsoluteEpisodeNumberParserFixture : CoreTest
         [TestCase("Tokyo Ghoul: RE S2 - Episode 4 VOSTFR (1080p)", "Tokyo Ghoul RE S2", 4, 0, 0)]
         [TestCase("To Aru Majutsu no Index III - Episode 5 VOSTFR (1080p)", "To Aru Majutsu no Index III", 5, 0, 0)]
         [TestCase("[Prout] Steins;Gate 0 - Episode 5 VOSTFR (BDRip 1920x1080 x264 FLAC)", "Steins;Gate 0", 5, 0, 0)]
+        [TestCase("【喵萌奶茶屋】★10月新番★[哥布林杀手/Goblin Slayer][12END][720p][繁体][招募翻译校对]", "Goblin Slayer", 12, 0, 0)]
+        [TestCase("[爱恋字幕社][盾之勇者成名录][Tate no Yuusha no Nariagari][02][720P][GB][MP4]", "Tate no Yuusha no Nariagari", 2, 0, 0)]
```

Comment:
Which of these can Sonarr ACTUALLY process?

Reply:
Oh.. Just found that Sonarr will strip all non-ASCII charaters in release title